### ISSUE TITLE
Save dialog doesn't work on Web

### ISSUE THREAD

**Issue by emilk** [rerun-io/rerun#7138]
Rerun menu -> Save

![image](https://github.com/user-attachments/assets/939a7ac7-b54e-4cbd-957c-f63dac200307)

->

**Comment by jprochazk** [comment]
It works if you click on the `click here to download your file` text

**Comment by emilk** [comment]
Oh no… ok, that confused the hell out of me. We should make a PR to `rfd` to improve this.

What do you think about the possibility of being able to directly start the file download on the first click – what would be required for that?

**Comment by jprochazk** [comment]
> What do you think about the possibility of being able to directly start the file download on the first click – what would be required for that?

There was some PR in progress for that:
- https://github.com/PolyMeilex/rfd/pull/129

Looks like it stalled though

**Comment by jprochazk** [comment]
Posted my thoughts in the original issue: https://github.com/PolyMeilex/rfd/issues/128#issuecomment-2286581683

TL;DR: We could use better APIs in Chromium-based browsers, and otherwise fall back to the existing implementation
